package.json

```
@@ -0,0 +1,73 @@
+{
+  "name": "react-native-with-moxy",
+  "version": "0.0.1",
+  "description": "The https://{project-domain} web app",
```

I guess we can go with the following for the time being to be consistent with Next with MOXY:

```suggestion
  "description": "The https://play.google.com/store/apps/details?id=<package> Android and https://apps.apple.com/us/app/<name>/id<AppleID> iOS apps",
```